server/test/client-kv-test.c

```diff
@@ -7,7 +7,7 @@
 #include <unistd.h>
```

Comment:
It might be best to wait until #58 is ready before we merge this then, in order to avoid duplicates.